source_code/config/settings/local.py

```diff
@@ -74,13 +75,19 @@
 
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'qweryacademy',
-        'USER': 'postgres',
-        'PASSWORD': '123456',
-        'HOST': 'localhost',
-        'PORT': '',
-    }
-}
+db_from_env = dj_database_url.config()
```

Comment:
This isn't what was configured before. If you look at the top of this file, it is inheriting from `common`. in the `common.py` the `DATABASES` configuration is already configured and is expecting an environmental varibale `DATABASE_URL`. So for development, you just needed to set your envioronmental variable alone. You are not supposed to add any configuration to the `local.py` at all.